src/storage/hummock_test/src/compactor_tests.rs

```diff
@@ -625,34 +639,19 @@ pub(crate) mod tests {
         };
         // 2. get compact task and there should be none
         let compact_task = hummock_manager_ref
-            .manual_get_compact_task(
-                StaticCompactionGroupId::StateDefault.into(),
-                manual_compcation_option,
-            )
+            .manual_get_compact_task(compaction_group_id, manual_compcation_option)
             .await
             .unwrap();
 
         assert!(compact_task.is_none());
 
-        // 3. get the latest version and check
-        let version = hummock_manager_ref.get_current_version().await;
-        let output_level_info = version
-            .get_compaction_group_levels(StaticCompactionGroupId::StateDefault.into())
-            .levels
-            .last()
-            .unwrap();
-        assert_eq!(0, output_level_info.total_file_size);
-
-        // 5. get compact task
-        let compact_task = hummock_manager_ref
```

Comment:
Why do we remove this final check?

Reply:
After re-understanding the test semantics, I decided that using the get compact task to determine whether the data was cleaned or not was not necessary, so I went from using a simpler test sample

Reply:
After review the test semantics, I decided that using the get compact task to determine whether the data was cleaned or not was not necessary, so I went from using a simpler test case